metadata-ingestion/src/datahub/ingestion/source/sql/presto_on_hive.py

```diff
@@ -62,7 +76,32 @@ class PrestoOnHiveConfig(BasicSQLAlchemyConfig):
         description="Host URL and port to connect to. Example: localhost:3306",
     )
     scheme: str = Field(default="mysql+pymysql", description="", exclude=True)
+    metastore_db_name: Optional[str] = Field(
```

Comment:
Is it more apt to override the base `database` field definition with `metastore_db_name` as an alias here?

Reply:
if the user provides both `metastore_db_name` and `database` then:
* `database` is used to filter on the hive databases
* `metastore_db_name` is used to connect to the metastores's
so both are used and `metastore_db_name` is not an alias (in a programming sense)